.travis.yml

```diff
@@ -35,7 +35,7 @@ install:
   - docker tag "$IMAGE_TAG" "$IMAGE:$NPM_TAG"
 
 before_deploy:
-  - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
+  - docker login -u "$DOCKER_USERNAME" --password-stdin "$DOCKER_PASSWORD"
```

Comment:
shouldn't it be

    echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin

if it should be taken from stdin?

Reply:
thanks, changed